agent/api/container/container.go

```diff
@@ -190,6 +190,9 @@ type Container struct {
 	// KnownPortBindingsUnsafe is an array of port bindings for the container.
 	KnownPortBindingsUnsafe []PortBinding `json:"KnownPortBindings"`
 
+	// VolumesUnsafe is an array of volume mounts in the container.
+	VolumesUnsafe []docker.Mount
```

Comment:
correct me if im wrong: this ends up in the statefile yea? if yes - needs a statefile version bump?

Reply:
This field is not required in the statefile. Added json ignore tag.